vigetlabs/wordpress-site-starter
# [#58] Remove Core Blocks

# Summary

This PR removes some of the unused core blocks available by default in WordPress.

This PR also updates ACF Pro to the latest version. Once we have [deployment](https://github.com/vigetlabs/devops/issues/1514) set up, we'll configure the `auth.json` file and remove ACF Pro from the repository.

## Issues

* #58 

## Testing Instructions

1. Go to Pages > Add New
2. Ensure the following blocks are not listed in the block inserter.

* [Archives](https://wordpress.org/documentation/article/archives-block/) (`core/archives`)
* [Audio](https://wordpress.org/documentation/article/audio-block/) (`core/audio`)
* [Calendar](https://wordpress.org/documentation/article/calendar-block/) (`core/calendar`)
* [Custom HTML](https://wordpress.org/documentation/article/custom-html/) (`core/html`)
* [Latest Comments](https://wordpress.org/documentation/article/latest-comments-block/) (`core/latest-comments`)
* [Latest Posts](https://wordpress.org/documentation/article/latest-posts-block/) (`core/latest-posts`)
* [More](https://wordpress.org/documentation/article/more-block/) (`core/more`)
* [Page Break](https://wordpress.org/documentation/article/page-break-block/) (`core/nextpage`)
* [Preformatted](https://wordpress.org/documentation/article/preformatted-block/) (`core/preformatted`)
* [RSS](https://wordpress.org/documentation/article/rss-block) (`core/rss`)
* [Verse](https://wordpress.org/documentation/article/verse-block/) (`core/verse`)

## Screenshots

<img width="348" alt="Screenshot 2024-05-17 at 11 09 00 AM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/122309362/9553753d-d8fc-4da7-86d5-ca4dd4f652a4">
